### PR TITLE
implemented cards and column design

### LINKED ISSUE #19: Column and card block theme
- [ ] Implement column theme
- [ ] Implement card theme

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #19 

Test URLs:
- Before: https://main--fonterra-fernleaf--hlxsites.hlx.page/en/products
- After: https://19-column-and-card-block-theme--fonterra-fernleaf--hlxsites.hlx.live/en/products

### CODE REVIEW

**Comment by asthabh23** [comment]
@mkbansal1 , can you please point to the right test URLs where the implementation of blocks can be seen?
The test URLs should actually consist of URLs to the pages which are impacted by code in the particular PRs.

**Comment by mkbansal1** [comment]
@asthabh23  i have corrected the urls

**Comment by asthabh23** [comment]
Thanks @mkbansal1 , few suggestions:

- The product page isn't complete yet from the content perspective. 
- The product images aren't clickable yet, you would need to add that.
- The headings for the products need to be left aligned


Franklin Implementation:
Screenshot 2023-07-27 at 12 04 58 PM

Original site:

Screenshot 2023-07-27 at 12 07 34 PM

File: styles/styles.css [hunk 1]
```diff
@@ -250,21 +261,22 @@ main img {
...
+.section > div {
+  max-width: 61.25rem;
+  margin: 0 auto;
+  padding: 30px 0;
 }
 
-@media (min-width: 600px) {
-  main .section {
-    padding: 64px 32px;
+@media (max-width: 899px) {
```
Comment (asthabh23) [inline styles/styles.css:270]: I would prefer to have min-width media breakpoints

**Comment by mkbansal1** [comment]
- Image not clickable - this is authoring 
- title on left - center align give better experience, so i made it center aligned, but will make it left

File: styles/styles.css (hunk 1 above)
Comment (mkbansal1) [reply styles/styles.css]: done

**Comment by mkbansal1** [comment]
@asthabh23  I have made code fixes. Content can be updated later also.

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**